internal/component/otelcol/exporter/faro/faro.go

```diff
@@ -0,0 +1,108 @@
+package faro
+
+import (
+	"maps"
+	"time"
+
+	"github.com/grafana/alloy/internal/component"
+	"github.com/grafana/alloy/internal/component/otelcol"
+	otelcolCfg "github.com/grafana/alloy/internal/component/otelcol/config"
+	"github.com/grafana/alloy/internal/component/otelcol/exporter"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/faroexporter"
+	otelcomponent "go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/pipeline"
+)
+
+func init() {
+	component.Register(component.Registration{
+		Name:      "otelcol.exporter.faro",
+		Community: true,
```

Comment:
```suggestion
		Community: false,
```
This is not a community component?